composer.json

```diff
@@ -20,8 +20,6 @@
         "doctrine/common": "~2.4",
         "twig/twig": "~1.23|~2.0",
         "psr/log": "~1.0",
-        "symfony/security-acl": "~2.7",
```

Comment:
We can't AFAIK. There is no easy way to automate the knowledge that it should be reverted when merging in 3.0 IMO